ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -35,26 +35,46 @@ def toProtocolElement(self):
 
 
 class SimulatedPhenotypeAssociationSet(AbstractPhenotypeAssociationSet):
-    def __init__(self, parentContainer, localId, randomSeed):
+    def __init__(self, parentContainer, localId, randomSeed,
+                 numAssociations=2):
```

Comment:
Yes, that's exactly what I'm talking about.  Although there are a few other handoffs, in addition to that one, at lower levels.